File: cs/Iris/ExpressionElement/Instruction/AssignExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Iris.SimpleExpressionElement;

namespace Iris.ExpressionElement.Instruction
{
    public class AssignExpression : InstructionExpression
    {
        //public PathExpression path;
        public string name;
        public SimpleExpression value;

        public AssignExpression()
        {

        }

        public AssignExpression(string varName, SimpleExpression varValue)
        {

        }
    }
}

File: java/Iris/src/com/rokuan/iris/instruction/AssignExpression.java
package com.rokuan.iris.instruction;

import com.rokuan.iris.expression.SimpleExpression;

public class AssignExpression extends InstructionExpression
{
	//public PathExpression path;
	public String name;
	public SimpleExpression value;

	public AssignExpression()
	{

	}

	public AssignExpression(String varName, SimpleExpression varValue)
	{

	}
}
